src/toil_scripts/lib/files.py

```diff
@@ -78,3 +125,41 @@ def consolidate_tarballs_job(job, fname_to_id):
                         tarinfo.name = os.path.join(output_name, fname, os.path.basename(tarinfo.name))
                         f_out.addfile(tarinfo, fileobj=f_in_file)
     return job.fileStore.writeGlobalFile(out_tar)
+
+
+def untargz(input_targz_file, untar_to_dir):
+    """
+    Expands a tar.gz file
+
+    :param str input_targz_file:
+    :param str untar_to_dir: path to untar-ed directory/file
+    :return: path to untar-ed archive
+    :rtype: str
+    """
+    assert tarfile.is_tarfile(input_targz_file), 'Not a tar file.'
+    tarball = tarfile.open(input_targz_file)
+    return_value = os.path.join(untar_to_dir, tarball.getmembers()[0].name)
+    tarball.extractall(path=untar_to_dir)
+    tarball.close()
+    return return_value
+
+
+def get_files_from_filestore(job, work_dir, input_dict):
+    """
+    Copies files from the file store to a work directory. Will expand tar/gzipped files.
```

Comment:
I don't think it makes sense to roll these two functions (download file from file store, and expand) together. Many tools will run directly on GZIPed files, so it's not necessarily desirable to expand those.